### DISCUSSION TITLE
How to make the dashboard public ?

### DISCUSSION THREAD

**Discussion by KeziahMoselle** [electerious/Ackee#285] [Q&A] [answered]
I would like to remove the login section so the data becomes public, is it possible?

**Comment by electerious** [comment, marked as answer]
There're multiple tries to implement this feature, but it's currently not possible:

https://github.com/electerious/Ackee/issues/276
https://github.com/electerious/Ackee/pull/278
https://github.com/electerious/Ackee/pull/243
